package.json

```diff
@@ -137,8 +133,7 @@
     },
     "husky": {
         "hooks": {
-            "pre-commit": "check-clang-format \"yarn format\"",
-            "pre-push": "yarn bazel:format-check && yarn bazel:lint",
+            "pre-push": "check-clang-format \"yarn format\"",
```

Comment:
Does this include buildifier fixes? I think my biggest pain point from the last version was the different checks happening on commit and push.

Reply:
no, husky is no longer involved with buildifier at all